src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Browser SPA monitoring 2.0
+tags:
+  - Browser
+  - Single page app monitoring
+translate:
+  - jp
+metaDescription: The new browser SPA monitoring capability is a streamlined way to monitor your users' experience and app performance.
+freshnessValidatedDate: 02-01-2024
+---
+
+We're excited to announce an overhaul of our Pro+SPA agent, built to address numerous pain points voiced by our customers. This update tackles critical limitations of the current agent, paving the way for a smoother, more optimized, and future-proof SPA monitoring experience.
+
+<Callout variant="important">
+    This release is currently in optional trial mode for any browser customer who wants to explore the new features and provide your feedback. The current SPA experience will be gradually phased out as we prepare for a full rollout to all SPA monitoring customers.
```

Comment:
Something like restricted or limited preview atm; customers cannot opt in at this time. We decided to curate this list until further notice.

Reply:
If customers cannot opt in, how do they know if it's changing for them? Is just randomized for various customers?

Reply:
For APM customers, we work with select ones to enable their appl to get the flag needed to turn things on. For copy & paste customers, they make that config change themselves--no guardrails around this, but we know which customer is supported and who isn't.